content/source/docs/cloud/registry/design.html.md

```diff
@@ -7,19 +7,13 @@ description: |-
 
 # Using the Configuration Designer
 
-Terraform Cloud's private module registry includes a configuration designer, which can help you spend less time writing boilerplate code in a module-centric Terraform workflow.
+Terraform Cloud's private module registry includes a configuration designer that can help you spend less time writing boilerplate code in a module-centric Terraform workflow.
 
-The configuration designer is sort of like interactive documentation for your private modules, or very advanced autocompletion. It results in the same Terraform code you would have written in your text editor, but saves time by automatically discovering variables and searching module and workspace outputs for reusable values.
-
-## Workflow Summary
-
-The configuration designer lets you outline a configuration for a new workspace by choosing any number of private modules. It then lists those modules' variables as a fillable HTML form, with a helper interface for finding values that you can interpolate.
-
-Once you finish, the designer returns the text of a `main.tf` configuration, which you must copy and paste to create a new VCS repo for workspaces. The designer does not automatically create any repos or workspaces; it's only a shortcut for writing Terraform code faster.
+The configuration designer lets you outline a configuration for a new workspace by choosing any number of private modules. It then lists those modules' variables as a fillable HTML form, with a helper interface for finding values that you can interpolate. When you are finished, the designer returns the text of a `main.tf` configuration. This is the same Terraform code you would have written in your text editor.
 
 ## Accessing the Configuration Designer
 
-Navigate to the modules list with the top-level "Modules" button, then click the "+ Design Configuration" button in the upper right.
+Click the "Registry" button in the main navigation bar and then click "<> Design Configuration".
 
 ![Terraform Cloud screenshot: the design configuration button](./images/design-start.png)
 
```

Comment:
should we wrap "Select Modules" in quotes?

> The Select Modules page appears.

Reply:
Thank you for fixing everywhere :-) I see we do that on Learn.